robot_ids: ['0','1']
robot_types: {'0':2,'1':2}
robot_ranges: {'0':20,'1':20}
robot_map: [{'id': 0, 'comm_model': 'disc', 'connections': [1]}, {'id': 1, 'comm_model': 'disc', 'connections': [0]}]
topics: [{'name': 'allocated_point', 'pkg': 'gvgexploration', 'message_type': 'Frontier', 'message_pkg': 'gvgexploration', 'remap': 1, 'ns': 'robot'},{'name': 'auction_feedback', 'pkg': 'gvgexploration', 'message_type': 'Auction', 'message_pkg': 'gvgexploration', 'remap': 1, 'ns': 'robot'},{'name': 'base_pose_ground_truth', 'pkg': 'nav_msgs', 'message_type': 'Odometry', 'message_pkg': 'nav_msgs', 'remap': 1, 'ns': 'robot'},{'name': 'signal_strength', 'pkg': 'gvgexploration', 'message_type': 'SignalStrength', 'message_pkg': 'gvgexploration', 'remap': 1, 'ns': 'robot'},{'name': 'chosen_point', 'pkg': 'gvgexploration', 'message_type': 'ChosenPoint', 'message_pkg': 'gvgexploration', 'remap': 1, 'ns': 'robot'},{'name': 'auction_points', 'pkg': 'gvgexploration', 'message_type': 'Auction', 'message_pkg': 'gvgexploration', 'remap': 1, 'ns': 'robot'},{'name': 'initial_data', 'pkg': 'gvgexploration', 'message_type': 'BufferedData', 'message_pkg': 'gvgexploration', 'remap': 1, 'ns': 'robot'}]
topic_remaps: [{'name': 'allocated_point', 'pkg': 'geometry_msgs', 'message_type': 'Frontier', 'message_pkg': 'geometry_msgs', 'remap': 1, 'ns': 'robot'},{'name': 'auction_feedback', 'pkg': 'gvgexploration', 'message_type': 'Auction', 'message_pkg': 'gvgexploration', 'remap': 1, 'ns': 'robot'},{'name': 'karto_out', 'pkg': 'nav2d_karto', 'from': 'karto_out', 'to': '/karto_out', 'message_type': 'LocalizedScan', 'message_pkg': 'nav2d_msgs', 'remap': 1}]
shared_topics: {'0':{'1': {'initial_data':'BufferedData','auction_points':'Auction','auction_feedback':'Auction','allocated_point':'Frontier','chosen_point':'ChosenPoint'}},'1':{'0': {'initial_data':'BufferedData','auction_points':'Auction','auction_feedback':'Auction','allocated_point':'Frontier','chosen_point':'ChosenPoint'}}}

# for a different project
map_image_path: '$(find gvgexploration)/world/basic.png'
world_center: [40.0,15.0]
world_scale: 10
map_pose: [39.85,14.55]
